resources/user_config.ts

```diff
@@ -242,12 +298,11 @@ class UserConfig {
               variableTracker[field] = options[field];
             }
 
-            if (this.userFileCallbacks[overlayName])
-              this.userFileCallbacks[overlayName](jsFile, localFiles, options, basePath);
+            this.userFileCallbacks[overlayName]?.(jsFile, localFiles, options, basePath);
           } catch (e) {
             // Be very visible for users.
             console.log('*** ERROR IN USER FILE ***');
-            console.log(e.stack);
+            console.log(e);
```

Comment:
This is the only (intended) logic change.  TypeScript doesn't really support error types very well and `console.log(e)` covers this just as well.

Reply:
What about `(e as Error).stack` ?

Reply:
Yeah, I think that would work, but ultimately there's no interesting stack here (as compared to a runtime error).  The only stack is just "you're in the middle of loading a user file".